app/mutations/useArchiveMutation.ts

```diff
@@ -39,7 +40,10 @@ export default function useArchiveMutation<
   ) => Disposable,
   boolean
 ] {
-  const [updateMutation, isInFlight] = useMutation(mutation);
+  const [updateMutation, isInFlight] = useMutationWithErrorMessage(
+    mutation,
+    () => "An error occurred"
```

Comment:
I wasn't sure about the error message here

Reply:
Yeah this hook is generic. Let's leave it as-is for now